src/features/swap/components/SwapOrderConfirmationView/index.tsx

```diff
@@ -107,8 +117,32 @@ export const SwapOrderConfirmationView = ({ order, settlementContract }: SwapOrd
           ),
         ]}
       />
+
+      {isTwapOrder && (
+        <div className={css.partsBlock}>
+          <DataTable
+            rows={[
+              <Typography key="title" variant="body1" className={css.partsBlockTitle}>
+                <strong>
+                  Order will be split in{' '}
+                  <span className={css.numberOfPartsLabel}>${order.numberOfParts} equal parts</span>
```

Comment:
```suggestion
                  <span className={css.numberOfPartsLabel}>{order.numberOfParts} equal parts</span>
```
<img width="652" alt="Screenshot 2024-06-25 at 14 56 18" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/92b1877d-c42b-454b-a040-d1c787d9790a">